tests/datasets_validation/test_enriched_datasets.py

```diff
@@ -0,0 +1,79 @@
+import great_expectations as ge
```

Comment:
looking great as a start! 

We need a method to systematize this, so we have a way to check all our datasets, but that will be done elsewhere.